RELEASE/scripts/autoscend/combat/auto_combat_ed.ash

```diff
@@ -584,7 +584,7 @@ string auto_edCombatHandler(int round, monster enemy, string text)
 		}
 	}
 
-	if (canUse($item[Tattered Scrap of Paper], false))
+	if (canUse($item[Tattered Scrap of Paper], false) && !(have_effect($effect[Everything Looks Green]) > 0))
```

Comment:
@Malibu-Stacey  can this if be deleted now that we have general free run support?